src/main/java/ch/njol/skript/lang/Variable.java

```diff
@@ -256,9 +256,24 @@ public boolean isSingle() {
 
 	@Override
 	public String toString(@Nullable Event e, boolean debug) {
-		if (e != null)
-			return Classes.toString(get(e));
-		return "{" + (local ? "_" : "") + StringUtils.substring(name.toString(e, debug), 1, -1) + "}" + (debug ? "(as " + superType.getName() + ")" : "");
+		StringBuilder stringBuilder = new StringBuilder()
+			.append("{");
+		if (local)
+			stringBuilder.append("_");
```

Comment:
this should probably use https://github.com/SkriptLang/Skript/blob/adb683196f44582251e44b998c4a0d88edd41606/src/main/java/ch/njol/skript/lang/Variable.java#L72